boards/bouffalolab/bl6/bl604e_iot_dvk/bl604e_iot_dvk-pinctrl.dtsi

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2021-2024 ATL Electronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <dt-bindings/pinctrl/bl604x-pinctrl.h>
+
+&pinctrl {
+	uart0_default: uart0_default {
+		group1 {
+			pinmux = <GPIO7_UART0_RX>,
+				 <GPIO16_UART0_TX>;
```

Comment:
pinctrl docs not aligned to what I see here

Reply:
I see, left over from old version. Fixed!